R/crosswalk.R

```diff
@@ -1,16 +1,19 @@
 #' crosswalk
 #'
 #' Crosswalk between different metadata fields used by different repositories,
-#' registries and archives. For more details see https://codemeta.github.io/crosswalk
-#' This function requires an internet connection to obtain the latest crosswalk table.
-#' This function essentially applies the crosswalk table shown by \code{\link{crosswalk_table}}
-#' to a given JSON metadata record.
+#' registries and archives. For more details see
+#' https://codemeta.github.io/crosswalk This function requires an internet
+#' connection to obtain the latest crosswalk table. This function essentially
+#' applies the crosswalk table shown by \code{\link{crosswalk_table}} to a given
+#' JSON metadata record.
 #' @param x a JSON list or file with data fields to be crosswalked
 #' @param from the corresponding column name from the crosswalk table.
 #' @param to the column to translate into, assumes "codemeta" by default
-#' @param codemeta_context the address or contents of codemeta context. Leave at default
+#' @param codemeta_context the address or contents of codemeta context. Leave at
+#'   default
```

Comment:
Thanks for all the excellent work on this.  

Note that we should probably document what the default is, particularly since it is no longer visible in the function's signature in the documentation